### DISCUSSION TITLE
::error::Pull request not found

### DISCUSSION THREAD

**Discussion by brad-getpassport** [nektos/act#1052] [Q&A] [answered]
Trying out **_act_** for the first time.
1) Installed via brew
2) cd into existing repo
3) ran `act` to setup a MEDIUM image
4) ran the following command:  `act pull_request -s GITHUB_TOKEN --defaultbranch main `

Get the following errors:
```
INFO[0001]   ☁  git clone 'https://github.com/CondeNast/conventional-pull-request-action' # ref=v0.1.2 
[Pull Request/Lint PR title                       ]   🐳  docker cp src=/Users/bpost/.cache/act/CondeNast-conventional-pull-request-action@v0.1.2/ dst=/var/run/act/actions/CondeNast-conventional-pull-request-action@v0.1.2/
[Pull Request/Lint PR title                       ]   🐳  docker exec cmd=[mkdir -p /var/run/act/actions/CondeNast-conventional-pull-request-action@v0.1.2/] user= workdir=
[Pull Request/Lint PR title                       ]   🐳  docker exec cmd=[node /var/run/act/actions/CondeNast-conventional-pull-request-action@v0.1.2/dist/index.js] user= workdir=
[Pull Request/Lint PR title                       ]   ❗  ::error::Pull request not found. Use pull request event to trigger this action
[Pull Request/Lint PR title                       ]   ❌  Failure - CondeNast/conventional-pull-request-action@v0.1.2
```

When I run the `act -l pull_request` I get the following:
```
Stage  Job ID  Job name                                               Workflow name  Workflow file     Events      
0      pr      Lint PR title                                          Pull Request   pull-request.yml  pull_request
1      check   Check Python ${{ matrix.python }} on ${{ matrix.os }}  Pull Request   pull-request.yml  pull_request
```

Ideas?

**Comment by catthehacker** [comment, marked as answer]
https://github.com/nektos/act/wiki/Beginner's-guide#using-event-file-to-provide-complete-event-payload

**Reply by brad-getpassport** [reply]
Doh!  Maybe add a link to the README for the guide ;-)
Sorry didn't see it in the wiki...
Thanks.

**Reply by catthehacker** [reply]
It's https://github.com/nektos/act#events in README
It will be linked to wiki once we're sure that wiki has better quality than current README

**Reply by catthehacker** [reply]
Also I think that action will not work properly with `act` since it uses GitHub API to look for the pull request that triggered it

**Reply by brad-getpassport** [reply]
okay thanks... trying to debug a pull_request action that seems to have stopped working so was hoping this could help...

**Reply by ipatch** [reply]
> https://github.com/nektos/act/wiki/Beginner's-guide#using-event-file-to-provide-complete-event-payload

that link is redirecting now. is there an updated link?

---

ahh, i guess that info was moved to the user guide section of the website.

https://web.archive.org/web/20230629084908/https://github.com/nektos/act/wiki/Beginner's-guide#using-event-file-to-provide-complete-event-payload
